### PR TITLE
Replace CodeReady Containers to CRC

### PR DESCRIPTION
**Pull request by praveenkumar**
This PR replace CodeReady Containers to CRC in cmd, pkg, images
directories and also update the README.

### CODE REVIEW

**Comment by praveenkumar** [comment]
This is mostly find and replace, if some of the string doesn't make sense we can modify accordingly.

File: pkg/crc/constants/constants_darwin.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 const (
 	OcExecutableName           = "oc"
 	PodmanRemoteExecutableName = "podman"
-	TrayExecutableName         = "CodeReady Containers.app"
+	TrayExecutableName         = "CRC.app"
```
Comment (praveenkumar) [inline pkg/crc/constants/constants_darwin.go:12]: @anjannath @gbraad I think we go with `crc.app` here?

File: pkg/crc/constants/constants_darwin.go [hunk 2]
```diff
@@ -23,5 +23,5 @@ func TrayExecutablePath() string {
 		return filepath.Clean(filepath.Join(version.InstallPath(), "..", "MacOS", "crc-tray"))
 	}
 	// Should not be reached, tray is only supported on installer builds
-	return filepath.Clean(filepath.Join(BinDir(), "CodeReady Containers"))
+	return filepath.Clean(filepath.Join(BinDir(), "CRC"))
```
Comment (praveenkumar) [inline pkg/crc/constants/constants_darwin.go:26]: @anjannath This should also change as per tray side changes..

**Review by gbraad — APPROVED** [review]

File: pkg/crc/constants/constants_darwin.go (hunk 1 above)
Comment (gbraad) [reply pkg/crc/constants/constants_darwin.go:12]: This should be `OpenShift Local.app`

File: pkg/crc/constants/constants_darwin.go (hunk 2 above)
Comment (gbraad) [reply pkg/crc/constants/constants_darwin.go:26]: The name tray remains `crc-tray.exe`, but internally named "Red Hat OpenShift Local"

**Review by gbraad — CHANGES_REQUESTED** [review]
Application name should be `OpenShift Local`

File: pkg/crc/constants/constants_darwin.go (hunk 2 above)
Comment (praveenkumar) [reply pkg/crc/constants/constants_darwin.go:26]: @gbraad This is not the file name but folder name.

Comment (gbraad) [reply pkg/crc/constants/constants_darwin.go]: yep... changed

**Review by gbraad — APPROVED** [review]
This is OK for the CLI

**Pull request merged**